plugins/metrics/app/controllers/metrics/application_controller.rb

```diff
@@ -12,5 +12,41 @@ def index
     def maia
       redirect_to "https://maia.#{current_region}.cloud.sap/#{@scoped_domain_name}?x-auth-token=#{current_user.token}"
     end
+
+    def gaas
+      grafana = {
+        "config": {
+          "ingressHost": "%s.grafana-svc.qa-de-1.cloud.sap",
+          "logLevel": "info",
+          "basicAuth": false,
+          "disableLoginForm": true,
+          "orgName": "sapcc",
+          "includeRolesInGroups": true,
+          "grafanaGroupRoleMap": "admin:Admin network_admin:Editor",
+          "authProxy": {
+            "enabled": true
+          }
+        }
+      }
+      uri = URI.parse("https://api.grafana-svc.#{current_region}.cloud.sap/api/v1/grafana")
+
+      header = {
+        "Content-Type": "application/json",
+        "x-auth-token": "#{current_user.token}"
+      }
+      https = Net::HTTP.new(uri.host, uri.port)
+      https.use_ssl = true
+      request = Net::HTTP::Post.new(uri.request_uri, header)
+      request.body = grafana.to_json
+    
+      # Send the request
+      response = https.request(request)
+      if response.code == '200' or response.code == '201'
+        body = JSON[response.body]
+        print body['hostname']
+        redirect_to "https://"+ body['hostname']
+      end
```

Comment:
todo: error handling!

Reply:
I am doing this now server side. It will return the url once the Grafana pod/ingress is up and running.

Reply:
Yes, sorry, I wrote this comment a while ago for an older version of the pull request. It's outdated now. I'm not sure why i posted this again with the review. Maybe I forgot to finish the review last time and my comment never showed up. My actual comment this time is the one about the policy entry :)